package/skelcd-control-openSUSE.changes

```diff
@@ -1,17 +1,32 @@
+-------------------------------------------------------------------
+Wed Jun 12 08:16:39 UTC 2019 - Simon Lees <sflees@suse.de>
+
+- Generic Desktop should also now install basic_desktop patterns
+  as it now pulls in icewm rather then X11 (boo#1124865)
+- Gnome, KDE and XFCE system roles now pull in enhanced_base as
+  they have always done so in the past.
+
+-------------------------------------------------------------------
+Wed Jun 12 07:16:39 UTC 2019 - Guillaume GARDET <guillaume.gardet@opensuse.org>
```

Comment:
This entry is being duplicated from earlier? That does nto seem like a changelog entry from YOUR changes

Reply:
I'm not sure whats happened here, I guess it got messed up merging, This commit was accepted while I was working on mine, i'll go fix it up.